index.html

```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="styles.css">
+  <title>Portfolio</title>
+</head>
+
+<body>
+  <header class="images">
+    <img src="images/MyLogo.png" alt="my logo">
```

Comment:
Kindly wrap this image in a `<a>` tag as the previous reviewer described 😃 I have explained it in the comment reply [above](https://github.com/Khadaro/Khadaro-Portfolio-Set-up-and-Mobile-First2/pull/1#discussion_r950359802).